### PR TITLE
Make concrete sensor IDL tests use standalone IDL files

### PR DESCRIPTION
**Pull request by Honry**
Concrete sensor APIs: Accelerometer, Gyroscope,
 Magnetometer, Orientation Sensor

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20009)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/294802155)

File: accelerometer/idlharness.https.html [hunk 1]
```diff
@@ -8,60 +8,32 @@
...
+  return fetch(url).then((response) => response.text());
+}
+
+promise_test(() => {
+  return Promise.all([
+    "/interfaces/dom.idl",
+    "/interfaces/generic-sensor.idl",
+    "/interfaces/accelerometer.idl",
+  ].map(fetchText)).then(doTest);
+}, "Test driver");
```
Comment (alexshalamov) [inline accelerometer/idlharness.https.html]: Should the name of the test reflect what is tested? Now it is "Test driver" for all sensors.

File: interfaces/orientation-sensor.idl [hunk 2]
```diff
@@ -0,0 +1,15 @@
+typedef (Float32Array or Float64Array or DOMMatrix) RotationMatrixType;
+
+[SecureContext, Exposed=Window]
+interface OrientationSensor : Sensor {
```
Comment (alexshalamov) [inline interfaces/orientation-sensor.idl]: Does idlharness check that constructor is not defined to verify that interface is not constructible?

File: accelerometer/idlharness.https.html (hunk 1 above)
Comment (Honry) [reply accelerometer/idlharness.https.html]: This is a standalone test for `promise_test` itself, I can use more appropriate name.

File: interfaces/orientation-sensor.idl (hunk 2 above)
Comment (Honry) [reply interfaces/orientation-sensor.idl]: Seems the answer is no, it is in TODO list at: https://github.com/w3c/web-platform-tests/blob/master/resources/idlharness.js#L1155

**Review by alexshalamov — APPROVED** [review]

**Comment by Honry** [comment]
@alexshalamov, thanks!

**Pull request merged**